README.md

```diff
@@ -104,13 +104,18 @@ repos:
       - id: terraform-fmt
       - id: terraform-validate
       - id: tflint
-      - id: gofmt
-      - id: goimports
-      - id: golint
       - id: golangci-lint
       - id: phony-targets
       - id: markdown-link-check
       - id: shellcheck
+
+      # The following hooks are redundant when golangci-lint is being. Our recommendation is to use golangci-lint
+      # as the main linter for go since it enables you to run all available linters in parallel.
+      # For details please see the example configuration https://github.com/mineiros-io/pre-commit-hooks/blob/master/.golangci.example.yml
+      # - id: gofmt
+      # - id: goimports
+      # - id: golint
+
```

Comment:
don;t think this comment is needed as it explained removed tools? why shoudl we maintain unneeded tooling?

Reply:
because this requires you to install golanglint ci as a dependency. A lot of the developers just really need gofmt, goimports and golint which can be executed without additional dependencies.